chainerx/__init__.py

```diff
@@ -1,8 +1,23 @@
 import os
 import warnings
 
-from chainerx import _build_info
-
+try:
+    from chainerx import _build_info
+except ImportError:
+    chainer_root_dir = os.path.abspath(__file__ + '../../..')
+    raise ImportError(
+        '''\
+Cannot import chainerx because _build_info.py cannot be found.
+
+The chainerx module being imported was not correctly installed by \
```

Comment:
How about "chainer and chainerx"?
Those who `import chainer` will also be affected, and they may have difiiculty interpret the message if it only mentions `chainerx`.